backend/config.py

```diff
@@ -27,44 +27,51 @@ class Config:
     )
     SQLALCHEMY_ENGINE_OPTIONS = {"pool_recycle": 299}
     SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    """jwt config"""
     JWT_BLACKLIST_ENABLED = True
 
     """mail config"""
-    MAIL_DEBUG = True  # always on, needed for error checking
-    MAIL_USE_TLS = True  # always on, for security (TM)
-    MAIL_USE_SSL = False  # always off, for security (TM)
-    MAIL_WEB_SERVER = ""  # defaults to empty str, might be set later
+    MAIL_DEBUG = True  # always on, for error checking
+    MAIL_USE_TLS = True  # always on, for security
+    MAIL_USE_SSL = False  # always off, for security
```

Comment:
Shouldn't SSL be good for security? :D

Reply:
good point, that's probably a mistake? will check that

Reply:
nope, probably not a mistake. It's a default from flask_mail and it seems to make sense. Ethereal doesn't work with SSL on (though is happy to use TLS option above). Maybe something with ports? Not tested with SES.
I'm happy to leave it as is unless you want me to check it again.